plugins/copr.py

```diff
@@ -463,21 +464,50 @@ def _download_repo(self, project_name, repo_filename, chroot=None):
 
         for line in f:
             if re.match(r"\[copr:", line):
-                repo_filename = os.path.join(self.base.conf.get_reposdir,
-                                             "_" + line[1:-2] + ".repo")
+                self.repo_filename = os.path.join(self.base.conf.get_reposdir,
+                                                  "_" + line[1:-2] + ".repo")
             break
 
         # if using default hub, remove possible old repofile
         if self.copr_url == self.default_url:
             # copr:hub:user:project.repo => _copr_user_project.repo
-            old_repo_filename = repo_filename.replace("_copr:", "_copr", 1)\
+            old_repo_filename = self.repo_filename.replace("_copr:", "_copr", 1)\
                 .replace(self.copr_hostname, "").replace(":", "_", 1).replace(":", "-")\
                 .replace("group_", "@")
             if os.path.exists(old_repo_filename):
                 os.remove(old_repo_filename)
 
-        shutil.copy2(f.name, repo_filename)
-        os.chmod(repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
+        shutil.copy2(f.name, self.repo_filename)
+        os.chmod(self.repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
+
+    def _runtime_deps_warning(self, project_name, chroot=None):
+        repo_file = ConfigParser()
+        repo_file.read(self.repo_filename)
+
+        if len(repo_file.sections()) == 1:
+            return
+
+        runtime_deps = ""
+        for section in repo_file.sections():
+            if section in self.repo_filename:
+                continue
+            runtime_deps += section + "\n"
+
+        msg = _("""
+The Copr project {0} uses the following repositories as runtime dependencies:
+{1}
+Do you want to keep them enabled?""".format(project_name, runtime_deps))
+
+        if self.base._promptWanted():
```

Comment:
I see that we already use this in `_ask_user`, but we should be careful about that since it is a private variable.

Reply:
So what do you propose to do with this?

Reply:
I would keep it as is. The function is pretty straightforward and likely will not change

    def _promptWanted(self):
        # shortcut for the always-off/always-on options
        if self.conf.assumeyes and not self.conf.assumeno:
            return False
        return True

I just brought it up in case you wanted to talk to the DNF team and tell them that it is a useful function, that we use, and if they would like to add it to their API. But I guess I can just ... @dmach ^^

Reply:
Might be better to @m-blaha for this :slightly_smiling_face: